[2023-01-25 23:14:50,559] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-25 23:14:50,573] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-01-25 23:14:50,573] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:14:50,573] {taskinstance.py:1377} INFO - Starting attempt 3 of 5
[2023-01-25 23:14:50,573] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:14:50,591] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Respuesta_Confirmacion_NASA> on 2023-01-02 00:00:00+00:00
[2023-01-25 23:14:50,597] {standard_task_runner.py:52} INFO - Started process 3437 to run task
[2023-01-25 23:14:50,600] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Respuesta_Confirmacion_NASA', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpgw119u31', '--error-file', '/tmp/tmpkuwg2n6c']
[2023-01-25 23:14:50,602] {standard_task_runner.py:80} INFO - Job 461: Subtask Respuesta_Confirmacion_NASA
[2023-01-25 23:14:50,683] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-02T00:00:00+00:00 [running]> on host fd9891b969b0
[2023-01-25 23:14:50,786] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Respuesta_Confirmacion_NASA
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-01-25 23:14:50,788] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-25 23:14:50,789] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 20 && echo "ConfirmaciÃ³n de la NASA, pueden proceder" > /tmp/response_20230102.txt']
[2023-01-25 23:14:50,800] {subprocess.py:85} INFO - Output:
[2023-01-25 23:15:10,810] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-25 23:15:10,856] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Respuesta_Confirmacion_NASA, execution_date=20230102T000000, start_date=20230125T231450, end_date=20230125T231510
[2023-01-25 23:15:10,877] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-01-25 23:15:10,880] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 3437. PIDs of all processes in the group: [3437]
[2023-01-25 23:15:10,880] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 3437
[2023-01-25 23:15:10,880] {process_utils.py:75} INFO - Process psutil.Process(pid=3437, status='terminated', exitcode=0, started='23:14:50') (3437) terminated with exit code 0
